content/tutorials/geodesic-getting-started.md

```diff
@@ -0,0 +1,98 @@
+
+### Geodesic Usage Patterns
+
+Let's talk about a few of the ways that one can run Geodesic. Our toolbox has been built to satisfy many use-cases, and each result in a different pattern of invocation:
+
+1. You can **run standalone** Geodesic as a standard docker container using `docker run`. This enables you to get started quickly, avoid fiddling with configuration, or run a one off commands using some of the built in tools.
+   1. Example: `docker run -it --rm --volume $HOME:/localhost cloudposse/geodesic:latest-debian --login` opens a bash login shell (`--login` is our Docker `CMD` here) in our Geodesic container.
+   1. Example: `docker run -it --rm --volume $HOME:/localhost cloudposse/geodesic:latest-debian -c "terraform version"` executes the `terraform version` command as a one off and outputs the result.
+1. You can **install** Geodesic onto your local machine using what we call the docker-pipe-bash pattern. Similar to above, this enables a quickstart process but supports longer lived usage as it creates a callable script on your machine that enables reuse.
+   1. Example: `docker run --rm cloudposse/geodesic:latest-debian | bash -s latest-debian` installs `/usr/local/bin/geodesic` on your local machine which you can execute repeatedly via simply typing `geodesic`
+1. Lastly, you can **build your own toolbox** on top of Geodesic. Want to provide additional packages to your team on top of what geodesic provides or want to customize the name of Geodesic to the name of your organization? This is simple to do by using Geodesic as your base image in your own Dockerfile, adding your own Docker `RUN` commands or overriding environment variables, and then building a new image which you distribute to your team. This is more advanced usage and we'll cover how to do this in a future how-to article.
+
+In this tutorial, we'll be running Geodesic standalone using `docker run` to allow us to get up and running quickly.
+
+## Tutorial
+
+### 1. Start the Geodesic Shell
+
+First, at your terminal, let's start up the Geodesic shell!
+
+```bash
+docker run -it --rm --volume $HOME:/localhost cloudposse/geodesic:latest-debian --login
+```
+
+There are a few things going on there, so let's break that down a bit:
+
+1. We're using `docker run` to start a new container using the image `cloudposse/geodesic:latest-debian` which is [hosted on Docker Hub](https://hub.docker.com/r/cloudposse/geodesic)
+1. We're using the `-it` flags to start an interactive terminal session.
```

Comment:
```suggestion
1. We're using the `-it` flags to start an interactive terminal session with a TTY.
```

Reply:
@osterman it's funny -- I worked on this session for a bit and was going to use TTY and link out to what it is, but then I realized it's somewhat confusing of a topic. My thoughts are that it might be just similar to refer to `-it` as an interactive terminal session then potentially confusing the reader on what a TTY is.

Reply:
hrm... ok, let's then say `(e.g. with a TTY)` 

Fwiw, TTY's just screwed up @danjbh in docker-compose trying to start a NodeJS app that depended on it. I'm old skool, so it's hard for me to calibrate what's confusing or not. To me, working on the console, TTY's come up all the time. Also, the `-it` is short for `--interactive --tty`.  I don't think we need to dumb-it-down.

Reply:
Got on board with your line of thinking after finding a good resource to link out to. 👍